plugins/convert/color/manual_balance.py

```diff
@@ -21,6 +21,21 @@ def process(self, old_face, new_face, raw_mask):
                 image[:, :, idx] = image[:, :, idx] * (1 + adjustment[idx])
 
         image = self.convert_colorspace(image * 255.0, to_bgr=True)
+
+        constrast = max(-126, int(round(self.config["constrast"] * 1.27)))
+        brightness = max(-126, int(round(self.config["brightness"] * 1.27)))
+
+        if constrast or brightness:
+            image = self.adjustContrast(image, constrast, brightness)
+        return image
+
+    def adjustContrast(self, image, contrast, brightness):
```

Comment:
snake case for functions (`adjust_contrast`)
Remove contrast, brightness from function call